modules/appeals_api/config/schemas/shared/v0/ssn.json

```diff
@@ -0,0 +1,13 @@
+{

```

Comment:
I added this new shared schema for SSN, since it looks like there will be more opportunities to use it as we move params from headers to bodies